src/parse-mockdb.js

```diff
@@ -559,6 +559,34 @@ function queryMatchesAfterIncluding(matches, includeClause) {
 }
 
 /**
+ * Sort query results if necessary
+ */
+function sortQueryresults(matches, order) {
+  // sort order
+  let sortKey = order;
+  let sortDir = 1;
+  const sortArray = [];
+  if (order.charAt(0) === '-') {
+    sortKey = order.substring(1);
+    sortDir = -1;
+  }
+  matches.forEach(item => {
+    const keyVal = item[sortKey];
+    let sortVal = keyVal;
+
+    // Attempt to handle dates, numbers and strings
+    const keyDate = new Date(keyVal);
+    if (keyDate !== 'Invalid Date' && !isNaN(keyDate)) {
+      sortVal = keyDate.getTime();
+    }
+    sortArray.push({ dataItem: item, sortVal });
+  });
+  sortArray.sort((a, b) => (a.sortVal - b.sortVal) * sortDir);
```

Comment:
Yes, I'll take another look at this when I have a minute and update the code and tests to properly handle strings...